com.unity.multiplayer.mlapi/Runtime/Timing/NetworkTickSystem.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using Unity.Profiling;
+using UnityEngine;
+
+namespace MLAPI.Timing
+{
+    public class NetworkTickSystem
+    {
+#if DEVELOPMENT_BUILD || UNITY_EDITOR
+        private static ProfilerMarker s_Tick = new ProfilerMarker($"{nameof(NetworkTimeSystem)}.Tick");
+#endif
+
+        /// <summary>
+        /// Special value to indicate "No tick information"
+        /// </summary>
+        public const int NoTick = int.MinValue;
```

Comment:
-2bn is a huge small number. what about 0? if 0 would mean infinite/unlocked then what about -1 instead of -2bn?

Reply:
My idea behind not using -1 was that I wanted to make sure that building a delta between a valid tick and the `NoTick` does not result in a reasonable value so that we can detect this mistake. Not sure if that makes too much sense in hindsight.